### PR TITLE
Added tags support in the player timeline

### PR DESCRIPTION
**Pull request by rifaideen**
@Yuyz0112 - In this PR, I have added tags support to display it in the `rrweb-player` timeline.
Related to [#81](https://github.com/rrweb-io/rrweb/issues/81#issuecomment-632707032)

### CODE REVIEW

**Comment by rifaideen** [comment]
@Yuyz0112 - do you have any comments?

File: src/Controller.html [hunk 1]
```diff
@@ -8,6 +8,13 @@
       ref:progress on:click="handleProgressClick(event)"
     >
       <div class="rr-progress__step" ref:step style="width: {percentage}"></div>
+        {#each getCustomEvents as event,i (i)}
+          <div
+            title="{event.name}"
+            style="width: 10px;height: 5px;position: absolute;top: 2px;transform: translate(-50%, -50%);background: {event.background};left: {event.position};"
```
Comment (Yuyz0112) [inline src/Controller.html:14]: how about move the static styles to CSS?

Comment (rifaideen) [reply src/Controller.html:14]: Actually I have tried a lot on moving this part to the css class and I have constantly observed strange behavior like whenever I move this to class i.e: `.tag` the final build does not includes the css. It breaks the component.

FYI: I am not much familiar with Svelte.

File: src/Controller.html [hunk 2]
```diff
@@ -99,6 +106,40 @@
...
+          return eventPosition.toFixed(2);
+        };
+
+         // loop through all the events and find out custom event.
+        events.forEach((event) => {
+          /**
+           * we are only interested in custom event and calculate it's position
+           * to place it in player's timeline.
+           */
+          if (event.type === 5) {
```
[the remark this answers is not in the document]
Comment (rifaideen) [reply src/Controller.html:131]: you mean `event.type === EventType.Custom`?

[the remark this answers is not in the document]
Comment (rifaideen) [reply src/Controller.html]: updated this check in recent commit.

File: src/Controller.html (hunk 1 above)
Comment (Yuyz0112) [reply src/Controller.html:14]: It's okay, I will give a try after merge

**Comment by Yuyz0112** [comment]
Thanks a lot!

**Comment by rifaideen** [comment]
Thanks for merging it :)

**Pull request merged**